tests/python/pants_test/backend/python/rules/test_python_fmt_integration.py

```diff
@@ -0,0 +1,70 @@
+# Copyright 2016 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from pants_test.pants_run_integration_test import PantsRunIntegrationTest, ensure_daemon
+from pants.util.contextutil import temporary_file_path
+from os.path import relpath
+
+class PythonFmtIntegrationTest(PantsRunIntegrationTest):
```

Comment:
Can we add one test which _does_ do formatting, e.g. by creating a temporary file in-line in the test?